src/test/ui/suggestions/return-bindings.rs

```diff
@@ -1,5 +1,3 @@
-// run-rustfix
-
```

Comment:
Could we keep some rustfix version? What made it no longer apply neatly? I usually split files like these into two, one for things that are rustfix and another for the ones that aren't.

Reply:
It wasn't that, @fee1-dead just didn't think that this was `MachineApplicable` worthy and I didn't feel strongly either way.

Reply:
I think there's a category between `MachineApplicable` and `MaybeIncorrect` that's like "probably right given no other options", which this suggestion falls into.